src/olympia/amo/cron.py

```diff
@@ -33,7 +33,7 @@ def days_ago(days):
     log.info('Collecting data to delete')
 
     logs = (
-        ActivityLog.objects.filter(created__lt=days_ago(180))
+        ActivityLog.objects.filter(created__lt=days_ago(365))
```

Comment:
Can we have this as a constant defined somewhere?

Reply:
[`3ea0d55` (#20229)](https://github.com/mozilla/addons-server/pull/20229/commits/3ea0d55d748cfa225e4f4d532e5751faa0650bca)